core/primitives/src/stateless_validation/stored_chunk_state_transition_data.rs

```diff
@@ -1,19 +1,22 @@
 use crate::challenge::PartialState;
 use borsh::{BorshDeserialize, BorshSerialize};
-use near_primitives_core::hash::CryptoHash;
+use near_primitives_core::{code::ContractCode, hash::CryptoHash};
 use near_schema_checker_lib::ProtocolSchema;
 
 use super::contract_distribution::CodeHash;
 
 /// Stored on disk for each chunk, including missing chunks, in order to
 /// produce a chunk state witness when needed.
-#[derive(Debug, Clone, PartialEq, Eq, BorshSerialize, BorshDeserialize, ProtocolSchema)]
+// TODO(#11099): Implement migration to combine these into single version
```

Comment:
I keep adding new versions, but this is fine, we can implement a migration to combine into a single version after launching the feature as the data to migrate is small.